Dockerfile

```diff
@@ -9,8 +9,11 @@ COPY requirements.txt /
 
 RUN pip3 install -r /requirements.txt
 
-# Get ST-Link drivers
-RUN apt-get install stlink-tools
+# Get ST-Link drivers, openocd and gdb
+RUN apt-get update && apt-get -y install \ 
+    stlink-tools \
```

Comment:
Can you please pin (i.e. install specific versions) the versions of all these? It's very important to have a deterministic setup. With the way this works right now, the docker env could be completely different from one day to the next if the packages get updated

Reply:
Verified resolved in https://github.com/concordia-fsae/containers